src/directLineStreaming.ts

```diff
@@ -0,0 +1,351 @@
+// In order to keep file size down, only import the parts of rxjs that we use
+
+import { BehaviorSubject } from 'rxjs/BehaviorSubject';
+import { Observable } from 'rxjs/Observable';
+import { Subscriber } from 'rxjs/Subscriber';
+import * as BFSE from 'botframework-streaming';
+
+import {
+  Activity,
+  ConnectionStatus,
+  Conversation,
+  IBotConnection,
+  Media,
+  Message
+} from './directLine';
+
+const DIRECT_LINE_VERSION = 'DirectLine/3.0';
+const MAX_RETRY_COUNT = 3;
+const refreshTokenLifetime = 30 * 60 * 1000;
+const timeout = 20 * 1000;
```

Comment:
can this be overridable by the caller?

Reply:
Not really - this is a pattern copied from directline.ts. We could think about this being overridable by the called but that can be in the next phase of improvements to this library.